Framework/Core/src/O2ControlHelpers.cxx

```diff
@@ -185,6 +185,43 @@ std::vector<RawChannel> extractRawChannels(const DeviceSpec& spec, const DeviceE
   return rawChannels;
 }
 
+bool isUniqueProxy(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::uniqueProxyLabel) != spec.labels.end();
+}
+
+bool shouldPreserveRawChannels(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::preserveRawChannelsLabel) != spec.labels.end();
+}
+
+bool isQcReconfigurable(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::qcReconfigurable) != spec.labels.end();
+}
+
+void dumpQcConfig(std::ostream& dumpOut, const DeviceExecution& execution, const DeviceSpec& spec, const std::string& indLevel)
+{
+  // get the argument `--config`
+  std::string configPath;
+  auto it = std::find_if(execution.args.begin(), execution.args.end(), [&](char* v) { if (v) { return strcmp(v, "--config") == 0;} else {return false;} });
```

Comment:
No need for `&` in the lambda captures.
You can also consider simplifying the lambda to `v != nullptr && strcmp(v, "--config") == 0`.